cap/views.py

```diff
@@ -0,0 +1,12 @@
+"""Theme blueprint in order for template and static files to be loaded."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint, current_app, render_template
+
+blueprint = Blueprint(
```

Comment:
IMHO you don't need it until you create custom URL rules. Since `template_folder` and `static_folder` are the same for CAP application created by factory function.